client/my-sites/stats/activity-log/index.jsx

```diff
@@ -193,14 +218,16 @@ class ActivityLog extends Component {
 			startDate,
 		} = this.props;
 
-		const startOfMonth = moment( startDate ).startOf( 'month' ),
-			startOfMonthMs = startOfMonth.valueOf(),
-			endOfMonthMs = moment( startDate ).endOf( 'month' ).valueOf();
-		const filteredLogs = filter( logs, obj => startOfMonthMs <= obj.ts_site && obj.ts_site <= endOfMonthMs );
-		const logsGroupedByDate = map(
+		const siteOffset = this.getSiteOffsetFunc();
+
+		const startOfMonthMs = siteOffset( moment( startDate ).startOf( 'month' ) ).valueOf();
+		const endOfMonthMs = siteOffset( moment( startDate ).endOf( 'month' ) ).valueOf();
```

Comment:
In this case, `startDate` is a prop which is the first day of the month. If we apply the site offset to it, it can actually slip out of the desired month, messing up the month filtering! So we want to first make sure we have the start and end of the desired month before applying the site offsets to compare with our site offset timestamps.

It occurs to me now that since we are applying offsets to both the month start/end and the timestamps to determine which events fall in the current month, we could get the same effect by ignoring the offset completely for this operation.